plugin/entryBase.go

```diff
@@ -74,6 +76,19 @@ func (e *EntryBase) Attr(ctx context.Context) (Attributes, error) {
 	}, nil
 }
 
+func (e *EntryBase) cname() string {
+	return strings.Replace(
```

Comment:
This could also be `ReplaceAll`. That seems more clear.

Reply:
That's in Go 1.12 unfortunately